src/core/apiClientInstance.ts

```diff
@@ -25,7 +15,7 @@ export const apiClient = withSetupCheck(ApiClient, {
   },
 });
 
-export const reportsClient = withSetupCheck(ApiClient, {
+export const reportsClient = new ApiClient({
```

Comment:
It looks like after all the changes `apiClient` and `reportsClient` have exactly the same code. Do we need both separately?

Reply:
It's different endpoints, using different baseUrls, that's why we have separate Clients